packages/fxa-auth-server/lib/routes/account.ts

```diff
@@ -506,6 +559,17 @@ export class AccountHandler {
       tokenVerificationId,
     });
 
+    let keyFetchToken2;
+    if (password2) {
+      keyFetchToken2 = await this.createKeyFetchToken({
```

Comment:
@vbudhram @chenba - Maybe we should add a column to the keyFetchTokens table that indicates if the token is for a V1 or V2 password. I'm just thinking of in the future when we want to fully phase out V1 passwords, that having this bit of data might be helpful.

Reply:
I... half expected that we simply delete rows older than X from that table.  Are you thinking of using the version to find records to delete when phasing out v1 passwords?

Reply:
Nothing really comes to mind.